lib/data_update_scripts/20220105112823_migrate_logo_svg_data.rb

```diff
@@ -0,0 +1,38 @@
+module DataUpdateScripts
+  class MigrateLogoSvgData
+    def run
+      return if ::Settings::General.try(:logo_svg).blank? ||
+        (::Settings::General.try(:original_logo).present? && ::Settings::General.try(:resized_logo).present?)
+
+      logo_svg = Settings::General.logo_svg
+
+      # We do our best effort here: These are some (possibly not all) of the css variables that we may see in Forem
+      # logos. Hence, we try and replace them before converting.
+      logo_svg.gsub! "currentColor", "#090909"
+      logo_svg.gsub! "var(--base-inverted)", "#fff"
+      logo_svg.gsub! "var(--link-color)", "#3d3d3d"
```

Comment:
Please note that this is a best effort script and we try to cater for as many cases where the user may have hacked their logo with CSS variable to work differently within the light and dark theme. These css variables will fall away and are converted to static values. 

However, we are unable to cater for all cases and we just replacing the most common css variables. 

We will be giving creators the opportunity to change/tweak their logos before launch.